changelogs/fragments/sub_plugins_dir_change.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```

Comment:
If there is no backwards compatibility in the code below, this needs to be a `breaking_change`. You should also explain in more detail how users have to fix their own sub-plugins in their collections.